src/routes/Detail.js

```diff
@@ -34,6 +34,7 @@ import { ExternalLinkAltIcon } from '@patternfly/react-icons';
 import ReactJson from 'react-json-view';
 import { useQuery } from '../Utilities/hooks';
 import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+import CopyToClipboard from '../../../../react-json-view/react-json-view/src/js/components/CopyToClipboard';
```

Comment:
I don't think you need to import this thing. It's from outside of the project anyway.